azurelinuxagent/common/utils/shellutil.py

```diff
@@ -65,6 +65,7 @@ def has_command(cmd):
 
 def run(cmd, chk_err=True, expected_errors=[]):
     """
+    Note: Depracating in favour of `run_command()` function.
```

Comment:
Sorry for requesting this - could you add the reference of the runcommand: `azurelinuxagent.common.utils.shellutil.run_command`

Also, add a comment in `run_command`'s comments to tell that this is preferred over `run()`?

Also, any plans to search and replace this `run`?

Reply:
Done the changes you requested :)

 >Also, any plans to search and replace this run?

Since its not essential, we'll do it when we have more cycles available.